.generator/test_cli.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cli import handle_generate, handle_build, handle_configure
+
+
+def test_handle_configure_dry_run():
```

Comment:
Will you also have tests with `dry_run=False`? 

I could imagine that when `dry_run=False`, you could accumulate the commands that would be yielded by `dry_run=True` and pass them to a function `execute_run` that will execute them. In that case, the tests for `dry_run=False` need merely verify that the invocations all call `execute_run` with the expected parameters, and then we only need a single additional test for `execute_run` that it executes a given test command.